hedera-node/cli-clients/src/main/java/com/hedera/services/cli/signedstate/DumpFilesSubcommand.java

```diff
@@ -154,19 +233,28 @@ void reportSummary(
             @NonNull final Map<Integer, HederaFile> allFiles) {
         final var nTotalFiles = fileSummary.typeOccurrences().values().stream().reduce(0, Integer::sum);
 
-        final var nDataFiles = fileSummary.typeOccurrences().get(VirtualBlobKey.Type.FILE_DATA);
+        final var nDataFiles = fileSummary.typeOccurrences().getOrDefault(VirtualBlobKey.Type.FILE_DATA, 0);
         final var nDeletedDataFiles =
                 allFiles.values().stream().filter(HederaFile::isDeleted).count();
 
-        final var nMetadataFiles = fileSummary.typeOccurrences().get(VirtualBlobKey.Type.FILE_METADATA);
+        final var nMetadataFiles = fileSummary.typeOccurrences().getOrDefault(VirtualBlobKey.Type.FILE_METADATA, 0);
         final var nActiveDataFiles =
                 allFiles.values().stream().filter(HederaFile::isActive).count();
-        final var nActiveDataFilesWithMetadata =
-                allFiles.values().stream().filter(HederaFile::isActive).count();
+        final var nActiveDataFilesWithMetadata = allFiles.values().stream()
+                .filter(HederaFile::isActive)
+                .filter(hf -> null != hf.metadata())
```

Comment:
```suggestion
                .filter(hf -> hf.metadata() != null)
```
Personally I'd flip these to make it more readable, but that's a completely subjective suggestion

Reply:
So you don't like Yoda conditions?  It's an acquired taste to be sure.  (But you didn't complain about it up at L219-220?)

I just now tried `.filter(hf -> nonNull(hf.systemFileType())` (using `Objects::nonNull`) - I think I like it better and will use it.

(Actually what `Stream.filter` really needs is an overload `filter(Function<T,U> mapper, Predicate<U> predicate)` that does `e -> predicate(mapper(e))` so you could write, e.g., `filter(HederaFile::fileStore, isEqual(FileStore.SPECIAL))` - filtering on a condition of a field (or function) of the stream element but passing through the stream element.)

Reply:
This sparked an idea – a YouTube channel where Yoda reads source code as listeners fall asleep. You're welcome